Add howto docs for Cassandra Operators (issues#8189) (#9877)
mistercrunch/Airflow: docs/howto/connection/cassandra.rst

 .. Licensed to the Apache Software Foundation (ASF) under one
or more contributor license agreements.  See the NOTICE file
distributed with this work for additional information
regarding copyright ownership.  The ASF licenses this file
to you under the Apache License, Version 2.0 (the
"License"); you may not use this file except in compliance
with the License.  You may obtain a copy of the License at

 ..   http://www.apache.org/licenses/LICENSE-2.0

 .. Unless required by applicable law or agreed to in writing,
software distributed under the License is distributed on an
"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
KIND, either express or implied.  See the License for the
specific language governing permissions and limitations
under the License.

> Apache Cassandra Connection
> =======================

Reviewer: ```suggestion
Apache Cassandra Connection
===========================
```

> The Apache Cassandra connection type enables connection to Apache Cassandra.

Reviewer: ```suggestion
The Apache Cassandra connection type enables connection to ``Apache Cassandra <https://cassandra.apache.org/>``__.
```

Default Connection IDs
----------------------

Cassandra hook and Cassandra operators use ``cassandra_default`` by default.

Configuring the Connection
--------------------------
Host (required)

>     The host to connect to. Hosts can be provided in comma-separated string values.

Reviewer: ```suggestion
    The host to connect to. It is possible to specify multiple hosts as a comma-separated list.
```
This is not very clear since the first sentence says only one host.

Schema (required)

>     Specify the schema (keyspace) name to be used in the database.

Reviewer: ```suggestion
    The schema (keyspace) name to be used in the database.
```

Login (required)

>     Specify the user name to connect.

Reviewer: ```suggestion
    The user name to connect.
```

Password (required)

>     Specify the password to connect.

Reviewer: ```suggestion
    The password to connect.
```

Port (required)

>     Specify the port to connect.

Reviewer: ```suggestion
    The port to connect.
```

Extra (optional)

>     Specify the extra parameters (as json dictionary) that can be used in cassandra

Reviewer: ```suggestion
    The extra parameters (as json dictionary) that can be used in cassandra
```
